app/views/layouts/_footer.html.erb

```diff
@@ -10,7 +10,7 @@
           <div class="row">
             <% presenter.nav_links.each do |nav_link| %>
               <div class="col-md-4">
-                <%= link_to nav_link[:title], nav_link[:url], class: 'footer-column__link' %>
+                <%= link_to nav_link[:title], nav_link[:url], { class: 'footer-column__link' }.merge(presenter.add_target_blank?(nav_link[:title])) %>
```

Comment:
can this be computed inside `nav_link` and we can just use something like `nav_link[:attributes]`?

Reply:
yes, it can. I think that way might make the code cleaner.
Thank u !